src/commonMain/kotlin/teksturepako/pakku/api/actions/update/Update.kt

```diff
@@ -34,29 +35,36 @@ suspend fun updateMultipleProjectsWithFiles(
             }
     }
 
-    return@coroutineScope platforms.fold(projects.map { it.copy(files = mutableSetOf()) }.toMutableSet()) { acc, platform ->
+    val projectsToFiles = projects.associateTo(mutableMapOf()) { it.copy(files = mutableSetOf()) to it.files }
```

Comment:
Why is this needed? Can you do it without this mutable map?

Reply:
It's for finding the particular files from combined project which is different from the project in `projects`

Reply:
It's combined projects. Not new I think.

Reply:
Okay 👍